### PR TITLE
add sepolia & amoy network config

### PR DESCRIPTION
**Pull request by abbesBenayache**

### CODE REVIEW

File: config.dev.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "forwarderApiUrl": "https://forwarder.dev-oracle-factory.iex.ec",
   "onChainConfig": {
-    "5": {
-      "forwarder": "0x2aD6aD4F35cf7354fE703da74F459690dBcC12bf",
-      "oracle": "0x8dFf608952ADCDa4cF7320324Db1ef44001BE79b",
+    "80002": {
+      "forwarder": "0xc684E8645c8414812f22918146d72d1071E722AE",
+      "oracle": "0x36dA71ccAd7A67053f0a4d9D5f55b725C9A25A3E",
       "providerUrl": ""
```
Comment (PierreJeanjacquot) [inline config.dev.json:7]:
```
generic-oracle-dapp@2.1.1
└── ethers@5.6.8 
```
The version of ethers currently used does not provide fallback providers for networks 11155111 (Sepolia) and 80002 (Amoy)
While Sepolia was added in the latest `ethers@5` (5.7.2), Amoy was not, we still need to specify the `providerUrl`

TODO:
- update ethers dependency to the latest v5 `npm i ethers@5` (do not attempt to upgrade to `ethers@6` which cannot run in node 14.4.0)
- add `providerUrl` config for network `80002`

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]
The version of ethers currently used does not provide fallback providers for networks 11155111 (Sepolia) and 80002 (Amoy)

File: config.dev.json (hunk 1 above)
Comment (abbesBenayache) [reply config.dev.json]: commit : 996c96582550f7126ec95ff0631a398a517c967a

commit : a6305a464f2aff03ebac6e97f3f275136ed4f090

**Review by PierreJeanjacquot — APPROVED** [review]
LGTM

**Pull request merged**